controller/controller_server/csi_controller_server.py

```diff
@@ -246,14 +248,20 @@ def DeleteVolume(self, request, context):
                     logger.debug("Deleting volume {0}".format(vol_id))
                     array_mediator.delete_volume(vol_id)
 
-                except controller_errors.VolumeNotFoundError as ex:
+                except controller_errors.ObjectNotFoundError as ex:
                     logger.debug("volume was not found during deletion: {0}".format(ex))
 
                 except controller_errors.PermissionDeniedError as ex:
                     context.set_code(grpc.StatusCode.PERMISSION_DENIED)
                     context.set_details(ex)
                     return csi_pb2.DeleteVolumeResponse()
 
+        except controller_errors.ObjectIsStillInUseError:
+            logger.info("could not delete snapshot while in use: {0}".format(ex))
```

Comment:
snapshot?

Reply:
done

Reply:
object? why not volume?

Reply:
cause it can be snapshot too

Reply:
I thought we agreed that `DeleteVolume` should not try to delete any snapshot ever

Reply:
done, right